netquity/ios-fastlane
# Add AWS Deployment to Fastlane

Signed-off-by: Steven Mann <steven@cengtech.com>

# Summary of Change
Continuing on with what Joel had completed in this project:
1. I have added a step that will deploy build .ipa files for the iOS projects to our S3 buckets. From there we can distribute them to users.
2. I have added/ subtly changed some documentation within the README file.
3. I have made environment variables sourced from a locally defined (not git tracked) .env file

# Steps to Verify Changes
1. Make sure that you can set up the project identically as specified within the README.
2. Ensure that you can upload a build ipa file to the s3 bucket we use for deployments *ENSURE THAT THIS IS NOT A PRODUCTION LEVEL SCHEME - ONLY DO PHOBOS OR DEIMOS*
3. Ensure that the built app can be run on your iPad and that the Scheme you specified in the build step was correct. 

# Preconditions and Additional Details
Read the README, ensure you have the required environment variables specified, have your signing certificate added to your login keychain, and have specified the environment and schemes within a `config.yml` file

# Postconditions and Expected Outcome(s)
You are able to perform each step and the build present on your physical device operates correctly.
